AtSheet/explainer.md

```diff
@@ -83,7 +83,13 @@ This will import only the rules for `foo` - in this case, the `div { color: red;
 This will also import only this rules for "foo" - in this case, the `div { color: red; }` rule. This will *not* import any rules from `sheet.css` outside of "foo".
```

Comment:
```suggestion
This will also import only the rules for "foo" - in this case, the `div { color: red; }` rule. This will *not* import any rules from `sheet.css` outside of "foo".
```